app/javascript/App/bundles/NewsfeedPage/NewsListItem.tsx

```diff
@@ -44,9 +45,20 @@ const NewsListItem = (props) => {
     .replace(/<h1>/g, '')
     .replace(/<\/h1>/g, '')
 
+
+  let titleStyle: React.CSSProperties  = {};
```

Comment:
for consistency we're probably better off using one of these techniques, preferred listed first: 
1. styled components
2. css classes
3. inline styles

Reply:
Please, take a look at my last commit. I rewrote it with `classnames` and `css-modules`. What are you think about this way?